### PR TITLE
Default volume io1, fixes iops, gather rotated logs

### PR DESCRIPTION
**Pull request by pdmars**

### CODE REVIEW

File: aws/aws_instance.go [hunk 1]
```diff
@@ -188,9 +188,15 @@ func (awsI *Ec2Instance) runTerraformApply(volumeSize int, zookeeperSize int, ma
...
 	awsI.HTTPMask = mask
-	awsI.RootVolumeType = "standard"
 	awsI.RootVolumeSize = volumeSize
 
+	envVolType := os.Getenv("TF_VAR_root_volume_type")
+	if envVolType != "" && envVolType != "io1" {
+		awsI.RootVolumeIops = 0
+	} else {
+		awsI.RootVolumeIops = volumeSize * 50
+	}
```
Comment (buzztroll) [inline aws/aws_instance.go]: If the environment have both TF_VAR_root_volume_type=io1 and TF_VAR_root_volume_iops set the value for TF_VAR_root_volume_iops will be ignored and set to *50.  Is that expected?

File: aws/aws_volumes.go [hunk 2]
```diff
@@ -98,16 +95,14 @@ func (v *EbsVolumes) CreateSet(licensePath string, sizeOfEachVolume int, cluster
...
-		}
-	}
-	v.IoPs = fmt.Sprintf("%d", normalizedIOPS)
+
+	envVolType := os.Getenv("TF_VAR_stardog_home_volume_type")
+	if envVolType != "" && envVolType != "io1" {
+		v.IoPs = fmt.Sprintf("%d", 0)
+	} else {
+		v.IoPs = fmt.Sprintf("%d", sizeOfEachVolume * 50)
+	}
```
Comment (buzztroll) [inline aws/aws_volumes.go]: We have *50 twice now.  Should that be a global variable?

File: aws/aws_instance.go (hunk 1 above)
Comment (pdmars) [reply aws/aws_instance.go]: More or less. I figured we always just want the max allowed IOPS and that way the user doesn't have to do the math and try to set it correctly. If you want fewer IOPS, provision a smaller volume.

File: aws/aws_volumes.go (hunk 2 above)
Comment (pdmars) [reply aws/aws_volumes.go]: I guess it could be. Only will change if AWS changes that ratio.

**Pull request merged**